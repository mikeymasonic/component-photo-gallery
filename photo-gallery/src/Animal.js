import React, { Component } from 'react';

export default class Animal extends Component {
    render() {
        return <li>
            <div className="info-container">
                <h2>{this.props.animal.keyword}</h2>
                <p className="animal-title">{this.props.animal.title}</p>
            </div>

            <div className="image-container">
                <img
                    alt={this.props.animal.url}
                    src={this.props.animal.url} />
            </div>
            <p className="horns-class">Horns: {this.props.animal.horns}</p>
        </li>
    }
}

//this is my card - how you want it to look