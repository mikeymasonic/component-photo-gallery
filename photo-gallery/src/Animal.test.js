import React from 'react';
// import { render } from '@testing-library/react';
import Animal from './Animal';
import renderer from "react-test-renderer";
import animalData from "./data.js";
test('renders Animal.js correctly', () => {
  const tree = renderer
  .create(<Animal animal={animalData} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});