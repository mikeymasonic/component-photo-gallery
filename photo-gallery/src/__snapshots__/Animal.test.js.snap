// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Animal.js correctly 1`] = `
<li>
  <div
    className="info-container"
  >
    <h2 />
    <p
      className="animal-title"
    />
  </div>
  <div
    className="image-container"
  >
    <img />
  </div>
  <p
    className="horns-class"
  >
    Horns: 
  </p>
</li>
`;
